### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by Mutai-Gilbert**
In this pull request I was able to:
- Use validations for models.
- Write unit tests for models.

### CODE REVIEW

File: spec/models/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+describe Post, type: :model do
+  test_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                          post_counter: 0)
+  test_post = Post.create(author: test_user, title: 'Hello', text: 'This is my first post', comments_counter: 0,
+                          likes_counter: 0)
+
+  before { test_post.save }
+  it 'should have a title' do
+    expect(test_post.title).to eq 'Hello'
+  end
```
Comment (danmainah) [inline spec/models/post_spec.rb:12]: - [x] @Mutai-Gilbert great work adding a test for the title, however, you forgot to add a test to validate the title only accepts a title with 250 characters or less.

Comment (danmainah) [reply spec/models/post_spec.rb:12]: - [x] @Mutai-Gilbert great work adding tests for the post model, however, you forgot to add tests for the `increment_post_counter` custom method In post model.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Mutai-Gilbert , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Added validations for post and user model correctly
✔️ All tests for comment user and like are added and passing.

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: spec/models/post_spec.rb (hunk 1 above)
Comment (Mutai-Gilbert) [reply spec/models/post_spec.rb:12]: Done

Comment (Mutai-Gilbert) [reply spec/models/post_spec.rb:12]: Done.

Comment (Mutai-Gilbert) [reply spec/models/post_spec.rb:12]: Done

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @Mutai-Gilbert , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**